docs/manuals/core/concepts/scenario.md

```diff
@@ -20,9 +17,13 @@ first scenario. This process can be repeated across multiple scenarios.
 
     ![scenarios](../pic/scenarios.svg){ align=left }
 
-A scenario represents one instance of a business problem to solve. Each new business problem instance
-is represented by a new scenario. With Taipy, end-users can create, store, edit, and
-execute various scenarios within the same application.
+
+Two use-cases arise from the utilization of Taipy scenarios:
+
+### - Use case 1 : 
+Each _scenario_ represents a distinct instance of a business problem. 
```

Comment:
The "new" convention is to use stars (\*) to mark italics.
That's for a clearer support of Python symbols that can contain underscores, but not stars.